apptentive/src/main/java/com/apptentive/android/sdk/module/messagecenter/view/holder/WhoCardHolder.java

```diff
@@ -162,14 +164,14 @@ public void onClick(View view) {
 		}
 
 		// we need to properly announce the profile card: sending an implicit accessibility event
-		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
-			itemView.postDelayed(new Runnable() {
-				@Override
-				public void run() {
-						itemView.sendAccessibilityEvent(AccessibilityEvent.TYPE_ANNOUNCEMENT);
+		DispatchQueue.mainQueue().dispatchAsync(new DispatchTask() {
+			@Override
+			protected void execute() {
+				if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
+					itemView.sendAccessibilityEvent(AccessibilityEvent.TYPE_ANNOUNCEMENT);
```

Comment:
I think we should have a null check here.

Reply:
We can omit `null` check here